.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
We dont need any of the files in the .idea folder at the moment you can include them in a separate commit if the are needed later